src/app/accounts/destiny-account.ts

```diff
@@ -24,30 +24,22 @@ import { loggedOut } from './actions';
  * Platform types (membership types) in the Bungie API.
  */
 export const PLATFORM_LABELS = {
-  // t('Accounts.Xbox')
+  // t('Accounts.', {context: platform, contextList:'platforms'})
```

Comment:
I do not think this one is necessary, as I do not believe we show these strings anywhere anymore.